ui/apps/platform/src/Containers/Clusters/Components/AutoUpgradeToggle.tsx

```diff
@@ -8,7 +8,10 @@ import {
 } from 'services/ClustersService';
 
 function AutoUpgradeToggle(): ReactElement {
-    const [autoUpgradeConfig, setAutoUpgradeConfig] = useState<AutoUpgradeConfig>({});
+    const [autoUpgradeConfig, setAutoUpgradeConfig] = useState<AutoUpgradeConfig>({
+        enableAutoUpgrade: false,
+    });
```

Comment:
Here is a simpler change than what I suggested on Friday:

```ts
const [autoUpgradeConfig, setAutoUpgradeConfig] = useState<AutoUpgradeConfig | null>(null);
```

Delete the following `useState` line and `finally` block.

Edit condition of `if` statement below.
```ts
    if (!autoUpgradeConfig) {
        return <></>;
    }
```

Reply:
Done